packages/clerk-js/src/ui/elements/ProfileCard/ProfileCardContent.tsx

```diff
@@ -42,6 +43,7 @@ export const ProfileCardContent = (props: ProfileCardContentProps) => {
         borderColor: t.colors.$neutralAlpha50,
         boxShadow: t.shadows.$cardContentShadow,
       })}
+      id={PROFILE_CARD_SCROLLBOX_ID}
```

Comment:
Instead of passing around a ref, I added a targetable id where we needed to portal the drawer. Let me know if anyone has a better solution.